lapis2/src/main/kotlin/org/genspectrum/lapis/openApi/Schemas.kt

```diff
@@ -79,7 +81,12 @@ If none if provided in the request, LAPIS will generate one.
 
 const val DOWNLOAD_AS_FILE_DESCRIPTION = """
 Set to true to make your browser trigger a download instead of showing the response content by setting the
-'Content-Disposition' header to 'attachment'.
+'$CONTENT_DISPOSITION' header to 'attachment'.
+"""
+
+const val COMPRESSION_DESCRIPTION = """
+Optionally set this to return the response compressed in the specified format.
+Alternatively, you can set the '$ACCEPT_ENCODING' header to the respective value.
```

Comment:
When I try the compression of gzip in the browser I get the same json back. Is this intended? If so, it is strange to observe no change, when using the swagger. Maybe we could add something to the description?

Reply:
What happens if you additionally set `downloadAsFile=true`? Is it showing or saving an uncompressed file (i.e., it downloads and then directly uncompresses)?

Reply:
The browser automatically handles gzip compressed data. From what I observed, most requests that a browser issues have `Accept-Encoding: gzip, deflate, br` set, and the response is decompressed without a user noticing it. Is that what you mean?

Reply:
Is it possible to allow the user to get an actual .gz file downloaded to their disk using the browser?